lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -77,6 +73,8 @@ def self.auto_generate_changelog(repo_name, github_token, rate_limit_sleep)
       end
 
       def self.increase_version(current_version, type_of_bump, snapshot)
```

Comment:
Maybe rename to `calculate_next_version`? Since it won't always increase the version number now.